File: Chapter09/airline-payments/src/main/java/springfive/airline/airlinepayments/infra/web/WebClientProducer.java
package springfive.airline.airlinepayments.infra.web;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.reactive.function.client.WebClient;

@Configuration
public class WebClientProducer {

    @Bean
    public WebClient webClient(){
        return WebClient.create();
    }

}

File: Chapter04/tweet-gathering/src/main/kotlin/springfive/twittergathering/infra/web/WebClientProducer.kt
package springfive.twittergathering.infra.web

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.reactive.function.client.WebClient

/**
 * @author claudioed on 17/12/17.
 */
@Configuration
open class WebClientProducer {

    @Bean
    open fun webClient(): WebClient? {
        return WebClient.create()
    }

}